OneCommunityGlobal/HighestGoodNetworkApp
# Hem - Moved the tests

(PRIORITY High) Jae: Move Unit Tests
Some unit tests were put in the wrong folder and we need to fix that so everything is standardized.
What’s needed is to move previously written tests from src/__tests__ to their relevant component folder as outlined in this unit testing guide:[ https://docs.google.com/document/d/1-iWj_1XJ_hFE0lRAAWvOfKtrkacT5HIvfWUu-6shQ08/edit?pli=1#heading=h.lz7htr6nv182](https://docs.google.com/document/d/1-iWj_1XJ_hFE0lRAAWvOfKtrkacT5HIvfWUu-6shQ08/edit?pli=1#heading=h.lz7htr6nv182)
Those tests that are already stored in their component folder need to be moved to a Component/__tests__ folder so everything is nice and neat. This should take about 3-5 hours.

# Description
The primary aim of this PR is to enhance the organization and structure of our testing suite by relocating unit tests to their respective component folders.

## Changes Made:
Test Relocation: All unit tests previously residing outside the component directories have been moved to their corresponding __tests__ folders within the components.
Re-testing: Ensured the functionality and accuracy of the relocated tests by executing them after the relocation process.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. do `npm test` to check if all the tests are working as in below screenshot image

Result:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/28252459/edceb241-283d-49e5-97fb-6e2b1f8c0eff)

## Videos of changes:
Before making changes
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/28252459/47b487f6-fdb3-46ba-9932-353dfebebe80

After making changes
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/28252459/58367869-95e1-47ad-baa1-a80a2c74a530
